src/react/workflow/ExpirationForm.tsx

```diff
@@ -164,14 +165,15 @@ export function ExpirationForm({ bucketList, locations, prefix = '' }: Props) {
                     );
 
                     return (
-                      <Select
-                        id="sourceBucket"
-                        onChange={({ value }) => onChange(value)}
-                        options={options}
-                        formatOptionLabel={renderSource(locations)}
-                        isOptionDisabled={(option) => option.disabled === true}
-                        value={result}
-                      />
+                      <Select id='sourceBucket' value={result} onChange={onChange}>
+                        {options &&
+                          options.map((o, i) => (
+                            <Option title={o.disabled ? 'Versioning is disabled' : ''} key={i} value={o.value}
```

Comment:
For expiration workflows this is not necessary
```suggestion
                            <Option key={i} value={o.value}
```

Reply:
Ok, I remove it